src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2612,6 +2612,51 @@ class AlterExtensionStatementSegment(BaseSegment):
     )
 
 
+class WrapperReferenceSegment(ansi.ObjectReferenceSegment):
+    """A Wrapper object reference."""
+
+    type = "wrapper_reference"
+    match_grammar: Matchable = Ref("SingleIdentifierGrammar")
+
+
+class HandlerReferenceSegment(ansi.ObjectReferenceSegment):
+    """A Handler object reference."""
+
+    type = "handler_reference"
+    match_grammar: Matchable = Ref("SingleIdentifierGrammar")
+
+
+class ValidatorReferenceSegment(ansi.ObjectReferenceSegment):
+    """A validator object reference."""
+
+    type = "validator_reference"
+    match_grammar: Matchable = Ref("SingleIdentifierGrammar")
+
+
+class CreateForeignDataWrapperStatementSegment(BaseSegment):
+    """A CREATE FOREIGN DATA WRAPPER Statement.
+
+    Docs: https://fdw.dev/catalog/
+    """
+
+    type = "create_foreign_data_wrapper"
+    match_grammar: Matchable = Sequence(
+        "CREATE",
+        "FOREIGN",
+        "DATA",
+        "WRAPPER",
+        Ref("WrapperReferenceSegment"),
```

Comment:
The code is fine, but I'm curious why we need to define `WrapperReferenceSegment` and the other two instead of just directly using `SingleIdentifierGrammar`? Would the formatting be any different? Is it worth maintaining the extra code?

Reply:
No reason, I've simplified it now.